istio/tests/Makefile

```diff
@@ -0,0 +1,27 @@
+KUBE_VERSION=v1.9.4
+NGINMESH_VERSION=0.6.0
+WRK_VERSION=3.1.0
+GIT_COMMIT=$(shell git rev-parse --short HEAD)
+TEST_IMAGE_NAME=bookinfo_testing
+REPO=gcr.io/$(GCLOUD_PROJECT)
+TAG=-$(NGINMESH_VERSION)-$(GIT_COMMIT)
+TAG_LATEST=$(NGINMESH_VERSION)
+DOCKER_PUSH = gcloud docker -- push
+
```

Comment:
separate into release and dev.  release should use nginmesh namespace.  gcp is only for dev.

Reply:
ok